src/docs/components/install-tabs.svelte

```diff
@@ -0,0 +1,13 @@
+<script lang="ts">
+	import { Tabs } from '$docs/components';
+</script>
+
+<Tabs tabs={['npm', 'yarn', 'pnpm']} let:tab>
```

Comment:
Is a separate component necessary for this? It's fine tho

P.S. Gonna add dynamic slot names to that Dominik Twitter thread

Reply:
I don't think it's _necessary_, but the syntax highlighting in the markdown gets a bit funky when you start adding `let`s and slots and such!